serving/docs/lmi/tutorials/trtllm_manual_convert_tutorial.md

```diff
@@ -143,7 +143,7 @@ parameters: {
 }
 ```
 
-Here is a table contains TRTLLM supported parameters with our suggestions. (up to 0.6.1)
+Here is a table contains TRTLLM supported parameters with our suggestions for version 0.6.1. For newer versions, please see the [TRT-LLM user guide](../user_guides/trt_llm_user_guide.md).
```

Comment:
The container we have with 0.6.1 is out of support at this point. Can we remove this section?